chart/flux/README.md

```diff
@@ -173,7 +173,7 @@ The following tables lists the configurable parameters of the Weave Flux chart a
 | Parameter                       | Description                                | Default                                                    |
 | ------------------------------- | ------------------------------------------ | ---------------------------------------------------------- |
 | `image.repository` | Image repository | `quay.io/weaveworks/flux`
-| `image.tag` | Image tag | `1.6.0`
+| `image.tag` | Image tag | `<VERSION>`
```

Comment:
Is this a variable for the reader, just so you don't have to keep having to change it?

Reply:
I want to avoid changing this every time, I've seen it used like this in other charts.